stl/inc/xstring

```diff
@@ -1305,7 +1301,7 @@ public:
             } || same_as<typename remove_reference_t<_Range>::traits_type, _Traits>)
     constexpr basic_string_view(_Range&& _Rng) noexcept(
         noexcept(_RANGES data(_Rng)) && noexcept(_RANGES size(_Rng))) // strengthened
-        : _Mydata(_RANGES data(_Rng)), _Mysize(_RANGES size(_Rng)) {}
+        : _Mydata(_RANGES data(_Rng)), _Mysize(static_cast<std::size_t>(_RANGES size(_Rng))) {}

```

Comment:
Generally, we don't need to `std::`-qualify typenames (except - as @timsong-cpp pointed out, when they are part of a conversion-function id that we're looking up in a user's class):
```suggestion
        : _Mydata(_RANGES data(_Rng)), _Mysize(static_cast<size_t>(_RANGES size(_Rng))) {}
```